generator/template/default/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# BASE
+FROM node:11-alpine as base
+
+LABEL maintainer="https://github.com/kaizendorks"
+
+RUN mkdir /home/node/app
+WORKDIR /home/node/app
+
+COPY client/package.json client/package-lock.json
+RUN npm install
+
+# BUILD STAGE
+FROM base as build
+
+COPY client/ .
+RUN npm run build
+
+# SERVE STAGE
+# TODO: shouldnt this be an nginx-like image and just copy the resulting "/dist" folder from the build?
```

Comment:
Good point, this one is only ~70MB which is pretty reasonable and doesn't require any extra config to work.

But we should investigate which approach would be recommend.

Reply:
Yeah, I would assume docker and compose files are still _to be completed_. Just wanted to get the basic plugin structure.

It works, if you create a new vue project with the cli, then you can `npm install --save-dev file:/path/to/this/repo` this plugin, and generate the files with `vue invoke dockue`